docs/sources/shared/configuration.md

```diff
@@ -1114,8 +1114,12 @@ dataobj:
       # CLI flag: -dataobj-consumer.sha-prefix-size
       [shaprefixsize: <int> | default = 2]
 
-    # The maximum amount of time to wait in seconds before flushing an object
-    # that is no longer receiving new writes
+    # The amount of time to wait to reach the target object size.
+    # CLI flag: -dataobj-consumer.flush-timeout
+    [flush_timeout: <duration> | default = 1h]
```

Comment:
Hm, we might want to think of a different name here, `flush_timeout` can be misinterpreted as the timeout for the request where we flush to object storage.

What about something like `max_age`?

Reply:
I thought a lot about this. We also have `idle-flush-timeout`, so for me the question was:

a. Do I also rename `idle-flush-timeout` so we have like `max-age` and `max-idle-age`.
b. Do I keep `idle-flush-timeout` and then have `flush-timeout`.

Do you think I should choose option a?